### ISSUE TITLE
disappearance of icons

### ISSUE THREAD

**Issue by ArnoStrasbourg** [gpailler/MMM-aqicn#6]
Hello,

I found a problem on my MM with the module. When it is activated, it works fine, but some icons of other modules disappear. For example :
In the calendar module : All icons disappear
In the SystemStats module : 3/5 icons disappear
In the jeedom module : All icons disappear except the first two

Any idea about this problem ? have you ever noticed that ?

**Comment by ArnoStrasbourg** [comment]
I just proceded of a new installation of MM. Put the modules one by one.
Same problem with AQICN. 
Module NETATMO also dissappear

@gpailler could you have a look on this issue ?

**Comment by gpailler** [comment]
Hi @ArnoStrasbourg 
Could you open de Developer Tools on the MM (Ctrl+Shift+I) and check if there is any error in the console?

**Comment by ArnoStrasbourg** [comment]
Hi Grégoire,
Here is the complete log

```
main.js:456 Initializing MagicMirror.
translator.js:202 Loading core translation file: translations/fr.json
translator.js:225 Loading core translation fallback file: translations/en.json
loader.js:179 Load script: modules/default/clock//clock.js
module.js:479 Module registered: clock
loader.js:150 Bootstrapping module: clock
loader.js:179 Load script: vendor/node_modules/moment/min/moment-with-locales.js
loader.js:179 Load script: vendor/node_modules/moment-timezone/builds/moment-timezone-with-data.js
loader.js:179 Load script: vendor/node_modules/suncalc/suncalc.js
loader.js:155 Scripts loaded for: clock
loader.js:194 Load stylesheet: modules/default/clock/clock_styles.css
loader.js:157 Styles loaded for: clock
loader.js:159 Translations loaded for: clock
loader.js:179 Load script: modules/default/calendar//calendar.js
module.js:479 Module registered: calendar
loader.js:150 Bootstrapping module: calendar
loader.js:233 File already loaded: moment.js
loader.js:155 Scripts loaded for: calendar
loader.js:194 Load stylesheet: modules/default/calendar/calendar.css
loader.js:194 Load stylesheet: vendor/css/font-awesome.css
loader.js:157 Styles loaded for: calendar
loader.js:159 Translations loaded for: calendar
loader.js:179 Load script: modules/MMM-OpenmapWeather//MMM-OpenmapWeather.js
module.js:479 Module registered: MMM-OpenmapWeather
loader.js:150 Bootstrapping module: MMM-OpenmapWeather
loader.js:233 File already loaded: moment.js
loader.js:155 Scripts loaded for: MMM-OpenmapWeather
loader.js:194 Load stylesheet: vendor/node_modules/weathericons/css/weather-icons.css
loader.js:194 Load stylesheet: modules/MMM-OpenmapWeather/MMM-OpenmapWeather.css
loader.js:157 Styles loaded for: MMM-OpenmapWeather
loader.js:159 Translations loaded for: MMM-OpenmapWeather
loader.js:179 Load script: modules/default/weatherforecast//weatherforecast.js
module.js:479 Module registered: weatherforecast
loader.js:150 Bootstrapping module: weatherforecast
loader.js:233 File already loaded: moment.js
loader.js:155 Scripts loaded for: weatherforecast
loader.js:233 File already loaded: weather-icons.css
loader.js:194 Load stylesheet: modules/default/weatherforecast/weatherforecast.css
loader.js:157 Styles loaded for: weatherforecast
loader.js:159 Translations loaded for: weatherforecast
loader.js:179 Load script: modules/MMM-aqicn//MMM-aqicn.js
module.js:479 Module registered: MMM-aqicn
loader.js:150 Bootstrapping module: MMM-aqicn
loader.js:179 Load script: //cdnjs.cloudflare.com/ajax/libs/jquery/2.2.2/jquery.js
loader.js:179 Load script: modules/MMM-aqicn/String.format.js
loader.js:155 Scripts loaded for: MMM-aqicn
loader.js:194 Load stylesheet: modules/MMM-aqicn/MMM-aqicn.css
loader.js:194 Load stylesheet: https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css
loader.js:157 Styles loaded for: MMM-aqicn
translator.js:173 MMM-aqicn - Load translation: translations/fr.json
/home/pi/MagicMirror/node_modules/electron/dist/resources/electron.asar/renderer/security-warnings.js:95 Electron Security Warning (Insecure Resources) This renderer process loads resources using insecure
  protocols.This exposes users of this app to unnecessary security risks.
  Consider loading the following resources over HTTPS or FTPS. 
 - http://localhost:8080/css/main.css
- http://localhost:8080/fonts/roboto.css
- http://localhost:8080/socket.io/socket.io.js
- http://localhost:8080/vendor/node_modules/nunjucks/browser/nunjucks.min.js
- http://localhost:8080/js/defaults.js
- http://localhost:8080/config/config.js
- http://localhost:8080/vendor/vendor.js
- http://localhost:8080/modules/default/defaultmodules.js
- http://localhost:8080/js/logger.js
- http://localhost:8080/translations/translations.js
- http://localhost:8080/js/translator.js
- http://localhost:8080/js/class.js
- http://localhost:8080/js/module.js
- http://localhost:8080/js/loader.js
- http://localhost:8080/js/socketclient.js
- http://localhost:8080/js/main.js
- http://localhost:8080/fonts/node_modules/roboto-fontface/fonts/roboto-condensed/Roboto-Condensed-Regular.woff2
- http://localhost:8080/translations/fr.json
- http://localhost:8080/translations/en.json
- http://localhost:8080/modules/default/clock//clock.js
- http://localhost:8080/vendor/node_modules/moment/min/moment-with-locales.js
- http://localhost:8080/vendor/node_modules/moment-timezone/builds/moment-timezone-with-data.js
- http://localhost:8080/vendor/node_modules/suncalc/suncalc.js
- http://localhost:8080/modules/default/clock/clock_styles.css
- http://localhost:8080/modules/default/calendar//calendar.js
- http://localhost:8080/modules/default/calendar/calendar.css
- http://localhost:8080/vendor/css/font-awesome.css
- http://localhost:8080/vendor/node_modules/@fortawesome/fontawesome-free/css/all.min.css
- http://localhost:8080/vendor/node_modules/@fortawesome/fontawesome-free/css/v4-shims.min.css
- http://localhost:8080/modules/MMM-OpenmapWeather//MMM-OpenmapWeather.js
- http://localhost:8080/vendor/node_modules/weathericons/css/weather-icons.css
- http://localhost:8080/modules/MMM-OpenmapWeather/MMM-OpenmapWeather.css
- http://localhost:8080/modules/default/weatherforecast//weatherforecast.js
- http://localhost:8080/modules/default/weatherforecast/weatherforecast.css
- http://localhost:8080/modules/MMM-aqicn//MMM-aqicn.js
- http://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.2/jquery.js
- http://localhost:8080/modules/MMM-aqicn/String.format.js
- http://localhost:8080/modules/MMM-aqicn/MMM-aqicn.css
  
 
For more information and help, consult
https://electronjs.org/docs/tutorial/security.
 This warning will not show up
once the app is packaged.
warnAboutInsecureResources @ /home/pi/MagicMirror/node_modules/electron/dist/resources/electron.asar/renderer/security-warnings.js:95
/home/pi/MagicMirror/node_modules/electron/dist/resources/electron.asar/renderer/security-warnings.js:145 Electron Security Warning (Insecure Content-Security-Policy) This renderer process has either no Content Security
    Policy set or a policy with "unsafe-eval" enabled. This exposes users of
    this app to unnecessary security risks.
 
For more information and help, consult
https://electronjs.org/docs/tutorial/security.
 This warning will not show up
once the app is packaged.
(anonymous) @ /home/pi/MagicMirror/node_modules/electron/dist/resources/electron.asar/renderer/security-warnings.js:145
translator.js:175 MMM-aqicn - Load translation fallback: translations/en.json
loader.js:159 Translations loaded for: MMM-aqicn
loader.js:179 Load script: modules/MMM-Netatmo-Thermostat//MMM-Netatmo-Thermostat.js
module.js:471 Check MagicMirror version for module 'MMM-Netatmo-Thermostat' - Minimum version:  2.1.0 - Current version: 2.11.0
module.js:473 Version is ok!
module.js:479 Module registered: MMM-Netatmo-Thermostat
loader.js:150 Bootstrapping module: MMM-Netatmo-Thermostat
loader.js:179 Load script: modules/MMM-Netatmo-Thermostat/q.min.js
loader.js:233 File already loaded: String.format.js
loader.js:179 Load script: modules/MMM-Netatmo-Thermostat/jquery-3.2.1.min.js
loader.js:155 Scripts loaded for: MMM-Netatmo-Thermostat
loader.js:194 Load stylesheet: modules/MMM-Netatmo-Thermostat/MMM-Netatmo-Thermostat.css
loader.js:157 Styles loaded for: MMM-Netatmo-Thermostat
translator.js:175 MMM-Netatmo-Thermostat - Load translation fallback: translations/en.json
loader.js:159 Translations loaded for: MMM-Netatmo-Thermostat
loader.js:179 Load script: modules/MMM-NewPIR//MMM-NewPIR.js
module.js:479 Module registered: MMM-NewPIR
loader.js:150 Bootstrapping module: MMM-NewPIR
loader.js:233 File already loaded: moment.js
loader.js:155 Scripts loaded for: MMM-NewPIR
loader.js:194 Load stylesheet: modules/MMM-NewPIR/MMM-NewPIR.css
loader.js:157 Styles loaded for: MMM-NewPIR
loader.js:159 Translations loaded for: MMM-NewPIR
loader.js:179 Load script: modules/MMM-SystemStats//MMM-SystemStats.js
module.js:479 Module registered: MMM-SystemStats
loader.js:150 Bootstrapping module: MMM-SystemStats
loader.js:233 File already loaded: moment.js
loader.js:179 Load script: modules/MMM-SystemStats/moment-duration-format.js
loader.js:155 Scripts loaded for: MMM-SystemStats
loader.js:233 File already loaded: font-awesome.css
loader.js:157 Styles loaded for: MMM-SystemStats
translator.js:173 MMM-SystemStats - Load translation: translations/fr.json
translator.js:175 MMM-SystemStats - Load translation fallback: translations/en.json
loader.js:159 Translations loaded for: MMM-SystemStats
loader.js:179 Load script: modules/MMM-Jeedom//MMM-Jeedom.js
module.js:479 Module registered: MMM-Jeedom
loader.js:150 Bootstrapping module: MMM-Jeedom
loader.js:155 Scripts loaded for: MMM-Jeedom
loader.js:233 File already loaded: font-awesome.css
loader.js:157 Styles loaded for: MMM-Jeedom
loader.js:159 Translations loaded for: MMM-Jeedom
loader.js:179 Load script: modules/MMM-Traffic//MMM-Traffic.js
module.js:479 Module registered: MMM-Traffic
loader.js:150 Bootstrapping module: MMM-Traffic
loader.js:155 Scripts loaded for: MMM-Traffic
loader.js:194 Load stylesheet: modules/MMM-Traffic/traffic.css
loader.js:233 File already loaded: font-awesome.css
loader.js:157 Styles loaded for: MMM-Traffic
loader.js:159 Translations loaded for: MMM-Traffic
loader.js:179 Load script: modules/MMM-GoogleCast//MMM-GoogleCast.js
module.js:479 Module registered: MMM-GoogleCast
loader.js:150 Bootstrapping module: MMM-GoogleCast
loader.js:155 Scripts loaded for: MMM-GoogleCast
loader.js:194 Load stylesheet: modules/MMM-GoogleCast/MMM-GoogleCast.css
loader.js:157 Styles loaded for: MMM-GoogleCast
translator.js:175 MMM-GoogleCast - Load translation fallback: translations/en.json
loader.js:159 Translations loaded for: MMM-GoogleCast
loader.js:179 Load script: modules/MMM-Fitbit2//MMM-Fitbit2.js
module.js:479 Module registered: MMM-Fitbit2
loader.js:150 Bootstrapping module: MMM-Fitbit2
loader.js:155 Scripts loaded for: MMM-Fitbit2
loader.js:194 Load stylesheet: modules/MMM-Fitbit2/MMM-Fitbit2.css
loader.js:157 Styles loaded for: MMM-Fitbit2
loader.js:159 Translations loaded for: MMM-Fitbit2
loader.js:194 Load stylesheet: css/custom.css
clock.js:45 Starting module: clock
calendar.js:77 Starting module: calendar
MMM-OpenmapWeather.js:117 Starting module: MMM-OpenmapWeather
weatherforecast.js:91 Starting module: weatherforecast
MMM-aqicn.js:22 Starting module: MMM-aqicn
MMM-SystemStats.js:44 Starting module: MMM-SystemStats
MMM-Jeedom.js:37 LOGMMM-Jeedom is started!
MMM-Jeedom.js:74 Array(8)
MMM-Traffic.js:43 Starting module: MMM-Traffic
MMM-GoogleCast.js:326 Starting module: MMM-GoogleCast
MMM-Fitbit2.js:121 Starting module: MMM-Fitbit2
main.js:474 All modules started!
module.js:198 MMM-OpenmapWeather is suspended.
module.js:198 weatherforecast is suspended.
main.js:280 Will not show weatherforecast. LockStrings active: module_3_weatherforecast
MMM-Traffic.js:301 received TRAFFIC_COMMUTE
module.js:205 clock is resumed.
module.js:205 MMM-OpenmapWeather is resumed.
module.js:205 MMM-aqicn is resumed.
module.js:205 MMM-Netatmo-Thermostat is resumed.
module.js:205 MMM-NewPIR is resumed.
module.js:205 MMM-SystemStats is resumed.
module.js:205 MMM-GoogleCast is resumed.
module.js:205 MMM-Fitbit2 is resumed.
api.openweathermap.org/data/2.5/forecast/daily?id=6450593&units=metric&lang=fr&APPID=e405a11441268263872f86698b9c1774:1 Failed to load resource: the server responded with a status of 401 (Unauthorized)
weatherforecast.js:266 weatherforecast: Your AppID does not support long term forecasts. Switching to fallback endpoint.
weatherRequest.onreadystatechange @ weatherforecast.js:266
MMM-Netatmo-Thermostat.js:132 Object
MMM-Fitbit2.js:102 Writing caloriesOut (data/goal): 1313/3000
MMM-Fitbit2.js:102 Writing floors (data/goal): 1/10
MMM-Fitbit2.js:102 Writing steps (data/goal): 970/10000
MMM-Fitbit2.js:102 Writing activeMinutes (data/goal): 0/60
MMM-Fitbit2.js:102 Writing distance (data/goal): 0.74/5
MMM-Fitbit2.js:102 Writing sleep (data/goal): 422/390
MMM-Fitbit2.js:102 Writing restingHeart (data/goal): 61/0
MMM-Fitbit2.js:106 Updating DOM
weatherforecast.js:257 [Violation] 'readystatechange' handler took 441ms
module.js:205 MMM-GoogleCast is resumed.
MMM-Traffic.js:301 received TRAFFIC_COMMUTE
```
Hope it helps.

**Comment by ArnoStrasbourg** [comment]
Today I have installed the roomTemperature module. With AQICN, the first icon disappear...

**Comment by gpailler** [comment]
I will test on a new MM install to try to reproduce. Unfortunately, the log you sent doesn't indicate any issue

**Comment by ArnoStrasbourg** [comment]
In parallel, I will test tonight on my MM with AQICN and only one of the other modules impacted

**Comment by ArnoStrasbourg** [comment]
Ok, done and I think I have a good new for you. I have only activated AQICN and SystemStats.
Icons are back.
So, I think the problem is not linked to AQICN. I will now reactivate modules one by one to see where the problem can be.

Edit : I have tested all my modules during more than 2 hours and I saw an interaction between AQICN and alerte and calendar modules : 
- When I activate Alerte and/or Calendar, AQICN activated, some icons disappear,
- When I activate Alerte and/or Calendar without AQICN, this icones are displayed,
- When I deactivate Alerte AND Calendar, AQICN activated, this icones are displayed

Test made with SystemStats.
I don't know if the problem comes from AQICN or both Calendar/Alert. But if you want to investigate, I hope this helps.

Edit 2 : in order to complete the test, I have installed the MMM-MyCalendar module. When It is activated with AQICN, unfortunatly icons disappear. Without AQICN, icons are back.

![20200514_105532](https://user-images.githubusercontent.com/6398055/81926552-6b166a00-95e2-11ea-8bca-d523f0b8efa2.jpg)

![20200514_105002](https://user-images.githubusercontent.com/6398055/81926592-76699580-95e2-11ea-983c-23881093da8b.jpg)

**Comment by ArnoStrasbourg** [comment]
@gpailler Do you looked at the problem a little bit ?

**Comment by gpailler** [comment]
Hi @ArnoStrasbourg ,

I just pushed the fix for the issue (9e616dad14b216a780aff633c48b6c2b588c1df5). It was a conflict with different font-awesome versions. Please update and check if it fixes your issue

**Comment by ArnoStrasbourg** [comment]
Hi @gpailler 

I have initiated the update in my MM-module and it work perfectly !

Thank you for your job.
